tools/bin/ecflow/processes/main/meteorological/chunk_forcings.py

```diff
@@ -0,0 +1,48 @@
+''' chunk_forcings.py
+usage: python save_metsim_by_year.py <metsim_config_file>
+Reads in output from MetSim (corresponding to information in
+metsim_config_file) and writes it to separate files by year for use in VIC '''
```

Comment:
Can MetSim directly output yearly data files?

Reply:
@arbennett tells me that running MetSim with the option `time_grouper` could work but that there is an increase in overhead in running MetSim because it runs each year separately instead of just altering the output.